### PR TITLE
Add compiler handler

### PR DESCRIPTION
**Pull request by peterNordin**
Adds generator for compiler command that should work with GCC and Clang
Only used in library import compilation for now. Will be extended to FMU export later

Related to #1711

### CODE REVIEW

File: HopsanGenerator/include/GeneratorTypes.h [hunk 1]
```diff
@@ -25,34 +25,73 @@
...
 
-#include "GeneratorUtilities.h"
+namespace hopsan {
 
+namespace os_strings {
+    constexpr auto win = "win";
+    constexpr auto win32 = "win32";
+    constexpr auto win64 = "win64";
+    constexpr auto apple = "apple";
+    constexpr auto Linux = "linux";
```
Comment (robbr48) [inline HopsanGenerator/include/GeneratorTypes.h]: Why capital letter on Linux but not on the others?

**Comment by peterNordin** [comment]
Also I will extend the code later, and use it for FMU export and such. I just realized that FMU import is not tested, it indirectly uses this code. We need a test for that as well.

File: HopsanGenerator/include/GeneratorTypes.h (hunk 1 above)
Comment (peterNordin) [reply HopsanGenerator/include/GeneratorTypes.h]: I will fix this later, the code needs some reworking anyway.

**Comment by peterNordin** [comment]
FMU import seems to be broken from Hopsan GUI (Linux), compile works but not loading

**Comment by peterNordin** [comment]
Fixed now (and tested), merging

**Pull request merged**